### PR TITLE
Fixes lints in lib/admin.php

### PR DESCRIPTION
**Pull request by belcherj**
#622

Fixes lints in lib/admin.php.  There is one remaining error but I do not have the knowledge to fix it.

### Detailed test instructions:

 - Not sure.

### CODE REVIEW

File: lib/admin.php [hunk 1]
```diff
@@ -155,6 +161,7 @@ function wc_admin_link_structure() {
 	array_unshift( $submenu['woocommerce'], $menu );
 
 	// Rename "Analytics" to Overview (otherwise this reads Analytics > Analytics).
+	// phpcs:ignore
```
Comment (claudiosanches) [inline lib/admin.php]: This should ignore everything, since we only want to allow overide of a WP global, better use `// WPCS: override ok.` instead.

Comment (claudiosanches) [reply lib/admin.php]: Like: 

```php
$submenu['wc-admin#/analytics'][0][0] = __( 'Overview', 'wc-admin' ); // WPCS: override ok.
```

File: lib/admin.php [hunk 2]
```diff
@@ -209,7 +226,11 @@ function wc_admin_admin_after_notices() {
 }
 add_action( 'admin_notices', 'wc_admin_admin_after_notices', PHP_INT_MAX );
 
-// TODO Can we do some URL rewriting so we can figure out which page they are on server side?
+/**
+ * TODO Can we do some URL rewriting so we can figure out which page they are on server side?
```
Comment (claudiosanches) [inline lib/admin.php]: Better include a properly docblock here, and then use `@todo`, we use this on core and will make easy to find TODOs later.

**Review by claudiosanches — CHANGES_REQUESTED** [review]
Looks good, but needs some small changes before we merge.

**Comment by belcherj** [comment]
All Fixed

**Review by claudiosanches — APPROVED** [review]
LGTM

**Pull request merged**